api/src/pdc/services/carpool/actions/CrosscheckAction.ts

```
@@ -26,68 +20,11 @@ import { getStatus } from "../helpers/getStatus.ts";
   ],
 })
 export class CrosscheckAction extends Action {
-  constructor(
-    private carpool: CarpoolRepositoryProviderInterfaceResolver,
-    private crosscheck: CrosscheckRepositoryProviderInterfaceResolver,
-    private identity: IdentityRepositoryProviderInterfaceResolver,
-    private config: ConfigInterfaceResolver,
-  ) {
+  constructor() {
     super();
   }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)